src/tokens/solana.tokenlist.json

```diff
@@ -65973,6 +65973,17 @@
       "tags": [
         "social-token"
       ]
+    },
+    {
+      "chainId": 101,
+      "address": "swimnKEr963p7EbCjsSnBCoYwytuZHPm3zbq6fKLHXb",
+      "symbol": "SWIM",
+      "name": "SWIM Protocol Token",
+      "decimals": 6,
+      "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/swimnKEr963p7EbCjsSnBCoYwytuZHPm3zbq6fKLHXb/logo.png",
+      "tags": [
+        "utility-token"
+      ]
```

Comment:
Not sure what this does but other projects add a lot of `extensions` here, we could add our Discord, Twitter, etc

Reply:
there's no official extensions spec according to the uniswap definition outside of a generic map object but i'll add some of the ones you mentioned. it would probably be up to each protocol/dApp to pull/display this data if they chose to do so

Reply:
added discord, telegram, medium, twitter, whitepaper and website links to `extensions`